python/pylibcudf/pylibcudf/tests/test_partitioning.py

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.
+
+import pyarrow as pa
+import pylibcudf as plc
+import pytest
+
+
+@pytest.fixture
+def pa_table():
+    return pa.table({"a": [1, 2, 3], "b": [1, 2, 5], "c": [1, 2, 10]})
+
+
+def test_partition(pa_table):
+    plc_result, result_offsets = plc.partitioning.partition(
+        plc.interop.from_arrow(pa_table),
+        plc.interop.from_arrow(pa.array([0, 0, 0])),
+        1,
+    )
+    pa_result = plc.interop.to_arrow(plc_result)
+    pa_expected = pa.table(
+        [[1, 2, 3], [1, 2, 5], [1, 2, 10]],
+        schema=pa.schema([pa.field("", pa.int64(), nullable=False)] * 3),
+    )
+    assert pa_result.equals(pa_expected)
```

Comment:
Is there a reason you aren't using the utilities we have for this like [`assert_column_eq`](https://github.com/rapidsai/cudf/blob/branch-24.10/python/pylibcudf/pylibcudf/tests/common/utils.py#L44)? You shouldn't need to do the arrow conversions manually. Not sure if this applies to other PRs that you currently have open as well. If it does, could you update the others as well.